dashboard/app/controllers/projects_controller.rb

```diff
@@ -280,6 +280,16 @@ def create_new
     )
   end
 
+  # GET /projects/for_level/:level_id
```

Comment:
A comment about this function's purpose and usage might be helpful for future visitors.

Reply:
added comments here and in `routes.rb`!